tests/integration/targets/server/tasks/test_primary_ips.yml

```diff
@@ -57,15 +57,13 @@
     that:
       - result is changed
 
-- name: cleanup server with primary ips
-  hetzner.hcloud.server:
-    name: "{{ hcloud_server_name }}"
-    state: absent
-
```

Comment:
Why did you remove this?

Reply:
Moved to end of tests to re-use server instead of creating a new one.
https://github.com/ansible-collections/hetzner.hcloud/blob/8534108d791776f22faae307dc3c860d3445ae59/tests/integration/targets/server/tasks/test_primary_ips.yml#L118C1-L121C18